build.gradle

```diff
@@ -253,6 +281,57 @@ integTest {
 
 testClusters.integTest {
     testDistribution = "ARCHIVE"
+
+    // Optionally install security
+    if (System.getProperty("security.enabled") != null) {
+        configurations.zipArchive.asFileTree.each {
+            plugin(provider(new Callable<RegularFile>() {
+                @Override
+                RegularFile call() throws Exception {
+                    return new RegularFile() {
+                        @Override
+                        File getAsFile() {
+                            return it
+                        }
+                    }
+                }
+            }))
+        }
+
+//        systemProperty("tests.opensearch.https", "true")
+//        systemProperty("tests.opensearch.username", "admin")
+//        systemProperty("tests.opensearch.password", "admin")
+
+        systemProperty("secure", "true")
+        systemProperty("username", "admin")
+        systemProperty("password", "admin")
+
+        extraConfigFile("admin-cert.pem", new File("$rootDir/src/test/resources/security/admin-cert.pem"))
```

Comment:
Have you explored to use `plugins.security.ssl.transport.keystore_filepath` instead of pem file? Wondering if `plugins.security.ssl.transport.keystore_filepath` could end up simpler setup?

Reply:
I tried a little bit, but was not able to get it to work. I dont see any keys in the https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchCluster.java#L581 where it the call is getting hung up. I reached out to security team to see if they have more info on it.